changelog.md

```diff
@@ -7,9 +7,15 @@
 * Renamed `FeederDirection.has` to `FeederDirection.contains`, which can be used via its operator version `in`. e.g. `BOTH.has(DOWNSTREAM)` can be replaced with
   `BOTH.contains(DOWNSTREAM)` or `DOWNSTREAM in BOTH`
 * Removed deprecated function `NetworkConsumerClient.getFeeder`.
+* Renamed function 'breadFirstSupplier' to 'breadthFirstSupplier' in 'BasicQueue' class.
+* Updated AssignToFeeders to support assigning AuxiliaryEquipment.
```

Comment:
I would call it breaking because we are changing the behavior of a publicly exposed and used interface.
After this merge any repo that is using the AssignToFeeders functionality will be running new code to process AuxilaryEquipment and ProtectionEquipment i.e. they will be getting a different result from running the exact same function on the exact same network (if that network had aux equipment or protection equipment).

We can make it backwards compatible / non breaking by adding an optional parameter to "enable auxiliary equipment and protection equipment processing" and having a default value of false for that parameter for example. That essentially leaves the functionality identical with the added enhancement that if you want you can choose to also process auxiliary equipment and protection equipment (I don't think we want this though, we want everyone to process aux/prot equipment and if they are not handling correctly for that code to be fixed).

Reply:
Hmm, I guess you could consider the result changing breaking, but that result isn't going to change in a breaking way - it'll just be new objects present in the containers, that should have been there anyway. I guess arguably it should under the Fixes section, because it's essentially a bug fix.

Reply:
Yep, I can see that line of reasonig, as in: "These things should have always been there"
I moved the entries to the Fixes section.